spot_msgs/srv/ExecuteDance.srv

```diff
@@ -1,4 +1,5 @@
 string choreo_file_content
+char[] choreo_sequence_bytes
```

Comment:
After some investigation the char primitive maps to a string and the bytes primitive maps to bytes. The SerializeToString call on protos returns a binary string so it's a lot cleaner to pass it through as an array of chars. Otherwise you have to take additional steps after the serialization to chop it into an array of bytes and then re-assemble on the other end.

For a fix, switched the name so it ties into the interaction with protobuf's serialization and left the type as char.

https://docs.ros.org/en/foxy/Concepts/About-ROS-Interfaces.html